code/src/terrat_ui_css/style.css

```diff
@@ -924,6 +924,37 @@
         @apply px-2;
     }
 
+    /* .billing-info { */
```

Comment:
This can be deleted if it's commented out

Reply:
done!